_providers/mail2tor.md

```diff
@@ -0,0 +1,21 @@
+---
+name: mail2tor
+status: OK
+domains:
+- mail2tor.com
+strict_tls: true
+server:
+- type: imap
+  hostname: g77kjrad6bafzzyldqvffq6kxlsgphcygptxhnn4xlnktfgaqshilmyd.onion
+  port: 143
+  socket: PLAIN
+- type: smtp
+  hostname: xc7tgk2c5onxni2wsy76jslfsitxjbbptejnqhw6gy2ft7khpevhc7ad.onion
+  port: 25
+  socket: PLAIN
+website: http://mail2torjgmxgexntbrmhvgluavhj7ouul5yar6ylbvjkxwqf6ixkwyd.onion/
+# website: http://mail2tor.com/
+last_checked: 2021-08-15
+---
+
+Email servers are only reachable over [Tor](https://www.torproject.org/).
```

Comment:
That line isn't shown in the web interface: https://testrun.org/provider/ca6be5c3e6c2f87e35a8e891612dc8a0536201b7/_site/mail2tor

Maybe we should adjust _includes/provider.html?